src/frontend/staff-portal/src/app/components/ticket-info/ticket-info.component.ts

```diff
@@ -635,11 +624,20 @@ export class TicketInfoComponent implements OnInit {
         this.initialDisputeValues.violationTicket.violationTicketCounts.forEach(violationTicketCount => {
 
           this.form.get('violationTicket').get('violationTicketCount' + violationTicketCount.count.toString()).patchValue(violationTicketCount);
-          this.form
-            .get('violationTicket')
-            .get('violationTicketCount' + violationTicketCount.count.toString())
-            .get('fullDescription')
-            .setValue(this.getLegalParagraphing(violationTicketCount));
+          if (!violationTicketCount.ticketedAmount)
+            this.form.get('violationTicket').get('violationTicketCount' + violationTicketCount.count.toString()).get('ticketedAmount').setValue(undefined);
+          if (violationTicketCount.section)
+            this.form
+              .get('violationTicket')
+              .get('violationTicketCount' + violationTicketCount.count.toString())
+              .get('fullDescription')
+              .setValue(this.violationTicketService.getLegalParagraphing(violationTicketCount) + " " + violationTicketCount.description);
+          else
+            this.form
+              .get('violationTicket')
+              .get('violationTicketCount' + violationTicketCount.count.toString())
+              .get('fullDescription')
+              .setValue(undefined);
```

Comment:
let fullDesc = violationTicketCount.section ? this.violationTicketService.getLegalParagraphing(violationTicketCount) + " " + violationTicketCount.description : undefined;
Then setValue